internal/generator/fft/template/domain.go.tmpl

```diff
@@ -45,37 +45,41 @@ type Domain struct {
 	cosetTableInv         []fr.Element
 }
 
-
-
-// NewDomain returns a subgroup with a power of 2 cardinality
-// cardinality >= m
-// shift: when specified, it's the element by which the set of root of unity is shifted.
-func NewDomain(m uint64, opts ...DomainOption) *Domain {
-	opt := domainOptions(opts...)
-	domain := &Domain{}
-	x := ecc.NextPowerOfTwo(m)
-	domain.Cardinality = uint64(x)
-
+// GetGenFrStar returns a generator of Z/rZ*
```

Comment:
Maybe just add a comment in the package doc what `r` is? When referring to package documentation then it is better to find instead of having to look at the implementation.

For "nothing-up-my-sleeve" transparency maybe should also add a comment about the property of the generators? Are we using smallest generators or do they have some other properties?

Reply:
The generators do not have any particular properties, they were found using Sage